backend/app/services/openapi/streaming.py

```
@@ -0,0 +1,228 @@
+                        content=[OutputTextContent(text=accumulated_text)],
+                    )
+                ],
+                previous_response_id=previous_response_id,
+            )
+            yield _format_sse_event(
+                "response.completed",
+                {"type": "response.completed", "response": final_response.model_dump()},
+            )
+
+        except Exception as e:
+            logger.exception(f"Error during streaming response: {e}")
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)